pages/toggle-button/permutations.page.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import ToggleButton, { ToggleButtonProps } from '~components/toggle-button';
+
+import createPermutations from '../utils/permutations';
+import PermutationsView from '../utils/permutations-view';
+import ScreenshotArea from '../utils/screenshot-area';
+
+const permutations = createPermutations<ToggleButtonProps>([
```

Comment:
Could we add coverage for using SVG and even URL for the icon?

Reply:
I think adding such examples does not bring much value, because:
1. Such examples are already provided in button permutation component, and toggle button inherits these props
2. The switching logic is the same as for icon name and it is already covered with u tests